### PR TITLE
Element 1.7.33

### PR DESCRIPTION
**Pull request by ghost**
thank you !

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.7.33-41 (2021/07/23)
+
+* Element 1.7.33
+
```
Comment (crazy-max) [inline CHANGELOG.md]: @HawkLiking Don't update the CHANGELOG

**Review by crazy-max — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
Sorry, I reverted the CHANGELOG

**Review by crazy-max — APPROVED** [review]
LGTM thanks!

**Comment by ghost** [comment]
(I just renamed the commit name from "Element 0.7.33" to "Element 1.7.33" sry)

**Pull request closed without merging**